client/overview/index.js

```diff
@@ -167,42 +171,42 @@ const OverviewPage = () => {
 			{ showConnectionSuccess && <ConnectionSuccessNotice /> }
 			{ ! accountRejected && ! accountUnderReview && (
 				<ErrorBoundary>
-					<>
-						{ showTaskList ? (
-							<>
-								<Card>
-									<Welcome />
-									<ErrorBoundary>
-										<TaskList
-											tasks={ tasks }
-											overviewTasksVisibility={
-												overviewTasksVisibility
-											}
-										/>
-									</ErrorBoundary>
-								</Card>
-								<Card>
-									<AccountBalances />
-								</Card>
-							</>
-						) : (
+					{ showTaskList && (
+						<Card>
+							<Welcome />
+							<ErrorBoundary>
+								<TaskList
+									tasks={ tasks }
+									overviewTasksVisibility={
+										overviewTasksVisibility
+									}
+								/>
+							</ErrorBoundary>
+						</Card>
+					) }
+					{
+						/* Show Payment Activity widget only when feature flag is set. To be removed before go live */
+						isPaymentOverviewWidgetEnabled && (
 							<Card>
-								<Welcome />
-								<AccountBalances />
+								<ErrorBoundary>
+									<PaymentActivity />
+								</ErrorBoundary>
 							</Card>
-						) }
-
-						<DepositsOverview />
-					</>
+						)
+					}
+					<Card>
+						<AccountBalances />
+					</Card>
```

Comment:
Note that the AccountBalanes should be moved back to the current position as per updated design, with the `PaymentActivity` card below.

Reply:
Thanks for the reminder 🙌 

Done here : https://github.com/Automattic/woocommerce-payments/pull/8412/commits/bd0278de6e046b068fc01f1d080a686a814574e9


![image](https://github.com/Automattic/woocommerce-payments/assets/4162931/fce6ea35-d0dd-40ce-99c2-40f236921896)

Reply:
👏 Nice one, although I also notice this has also inadvertently removed the `Welcome` message from showing when `showTaskList` is `false` – I think it should revert back to the `showTaskList ?` logic rather than `showTaskList &&`.

Reply:
Thanks for spotting this. It also drew my attention to the existing logic

* If there is a tasklist, group Welcome and Tasklist in one card, and keep Balance in separate card
* else Keep welcome and Balance in same card

Trying out the best way to do this.

Reply:
I have fixed this here - f9816f6 and f8f8e49